app/views/dorms/show.html.erb

```diff
@@ -171,7 +171,9 @@
           </div>
 
           <% if @student %>
-            <button id="pull-student" class="btn btn-primary btn-block">Pull Room as a Student</button>
+            <% if @period %>
```

Comment:
Could we do this change by modifying the original `if` to `if @student && @period`?  If `@period` is false but `@student` is true, it doesn't seem like we need the rest of this either.

Reply:
fixed